src/core/webidl.js

```diff
@@ -38,8 +38,20 @@ const templates = {
       : // Other keywords like sequence, maplike, etc...
         hyperHTML`<a data-xref-type="dfn" data-cite="WebIDL">${keyword}</a>`;
   },
-  reference(wrapped) {
-    return hyperHTML`<a data-xref-type="_IDL_">${wrapped}</a>`;
+  reference(wrapped, unescaped) {
+    let type = "_IDL_";
+    let cite = null;
+    switch (unescaped) {
+      case "Window":
+        type = "interface";
+        cite = "HTML";
```

Comment:
https://respec.org/xref/ only gives a singnle item for `Window`, so not sure why we need this...

Reply:
Ok, let’s remove 👍

Reply:
Removing it reintroduces the bug. Hard to understand...

Reply:
Magic 👏

Reply:
But seriously, let’s run with this and then deal with xref not matching separately as most specs are  currently in a broken state.

Reply:
Agreed.

Reply:
Will you publish this or should I do it instead?

Reply:
Sorry, just saw this... published. Still have the issue with Exposed=Worker... but it only affects a few specs. We can deal with it tomorrow.